rfcs/0102-moderation-team.md

```diff
@@ -0,0 +1,92 @@
+---
+
+# Summary
+[summary]: #summary
+
+Establish a team to perform moderation.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not
+sustainable to have to call on Graham any time there's a spammer or conflict
+that requires moderation, and we'd like to help the community become more
+self-regulating.
+
+(adopted from #98)
+
+# Detailed design
+[design]: #detailed-design
+
+The Moderation Team is a volunteer group that may receive, invite, and evaluate
+applications to the team or alter the composition at any time. The team's
+composition, contact information, and announcements should be maintained at
+[https://nixos.org/community/teams/moderation.html](https://nixos.org/community/teams/moderation.html).
+The team shall perform moderation activities on behalf of the community - with
+oversight via the RFC process - for discussions in [official project
+spaces](https://nixos.org/community/index.html) as well as unofficial spaces
+that seek and reach such an agreement with the team. The team should utilize
+the [NixOS Foundation mission](https://nixos.org/community/index.html) and the
+following statement during their duties:
+
+```
+The NixOS Foundation aims to promote participation without regard to gender,
+sexual orientation, disability, ethnicity, age, or similar personal
+characteristics. We want to strive to create and foster community by providing
+an intentionally welcoming and safe environment where all feel valued and cared
+for, and where all are given opportunity to participate meaningfully. The
+Foundation will work with the community in service of this goal.
+```
+
+ref: [twitter](https://twitter.com/grhmc/status/1390775249424338944)
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+The initial Moderation Team is defined to be @grahamc, @zimbatm, @domenkozar,
+and @ryantm.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+* The moderation team has limited guidance from this RFC on the processes and
+  procedures of the team.
+* This RFC is designed to address a narrow part of a current issue facing the
+  community. Additional RFCs may be needed to address additional concerns.
+* As this is a controversial topic there is a potential this RFC does not have
+  enough detail to be acceptable by the overall community.
+
+# Alternatives
+[alternatives]: #alternatives
+
+* An existing [RFC 98][].
+* Do nothing.
+
+# Unresolved questions
+[unresolved]: #unresolved-questions
+
+* Does the team require additional guidance?
+* Does the NixOS Foundation board want to be involved in this manner?
+
+# Future work
+[future]: #future-work
+
+* A potential RFC providing additional guidance and detail for the moderation
+  team's activities and functions.
+* The role of the moderation team could evolve through an effort similar to
+  [RFC 98][] into taking a broader community leadership responsibility as a
```

Comment:
98 is a complete non-starter for me and I don't think it should be used as any sort of official reference. It is an excessively long screed that injects a one-sided political view, and it's an unnecessarily large burden to make it a prerequisite for understanding what we're doing here. I suggest either dropping allusions to it or making it clear that this is summing up whatever ideas were being communicated there.

Reply:
Is the word _could_ & the fact that it's within the future section not concise enough?

I can understand your impressions on 98, but for the sake of bringing people together, we also wanted to leave a potential dovetail - subject of "(maybe) future work".

Reply:
> 98 is a complete non-starter for me and I don't think it should be used as any sort of official reference

Same view here. If this is supposed to be a simpler and less controversial version of RFC98, it should be completely standalone. Maybe incorporate some of its text, but not cite it altogether.

Reply:
I agree 98 is a non-starter, this was started as a direct response to the mess it became. Should we remove this reference and allow the PR's conversation+commentary to retain that historical link? (I don't have a clear strong preference either way, just trying to find what would be best and acceptable to as many people as possible.)

Reply:
We have to measure sentiments both ways. Consensus includes all stakeholders. And as much as 98 has turnes many participants sour, the authors of that RFC are by no means adverseries of this RFC.

I think certain amount of dovetailing [especially so in the future section] is neither 1) binding nor 2) ill-scoped.

If this is becoming a blocker, though. We might only retain it in spirit, not text.